django/db/models/aggregates.py

```diff
@@ -196,6 +245,52 @@ def _get_repr_options(self):
         return {**super()._get_repr_options(), "sample": self.function == "STDDEV_SAMP"}
 
 
+class StringAgg(Aggregate):
+    template = "%(function)s(%(distinct)s%(expressions)s%(order_by)s)%(filter)s"
+    function = "STRING_AGG"
+    name = "StringAgg"
+    allow_distinct = True
+    allow_order_by = True
+    output_field = TextField()
+
+    def __init__(self, expression, delimiter, **extra):
+        delimiter = Value(str(delimiter))
```

Comment:
We'll want to force the delimiter to be specified as a `Value`. This current form disallows using another expression as the delimiter such as a column reference.